### PR TITLE
feat: nginx will no longer run with root

### PR DESCRIPTION
**Pull request by cl-shaharshavit-rnd**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
-
 WORKDIR /opt/myapp
-
 COPY package*.json ./
-
 RUN yarn install --production
-
 COPY . .
-
 RUN yarn run confd:prod
```
Comment (asafMasa) [inline Dockerfile:6]: why to run the run confd:prod here?
just download the appropriate confd exe

Comment (asafMasa) [reply Dockerfile:6]: maybe create an nginx with confd for use in all clients and download service

File: Dockerfile [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
-
 WORKDIR /usr/share/nginx/html
-
 COPY --from=build /opt/myapp/build ./
-
 # Move env-config.js to location in which confd script expects it to be located
 RUN mkdir public && cp env-config.js public/env-config.js
 
+#chage nginx config to work without root
+RUN sed -i 's/listen       80;/listen       8080;/g' /etc/nginx/conf.d/default.conf  && \
```
Comment (asafMasa) [inline Dockerfile:21]: maybe create a nginx:1.19.1-alpine base package with this default.conf for use in all other clients

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (cl-shaharshavit-rnd) [reply Dockerfile:21]: we got only 2 nginx services and no registry ATM so I don't think it is relevant.
also if you want to add the permissions to there is can either be unclear to use or problematic to use due to lucking permissions

**Review by asafMasa — APPROVED** [review]

**Pull request merged**